source/blog/2017-11-07-fedora-atomic-27-on-multiarch.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Fedora 27 Atomic Host Availability on Multiple Architectures
+author: sinnykumari
+date: 2017-11-07
+published: true
+comments: false
+tags: atomic, fedora, multi-arch, ppc64le, aarch64
+published: true
+---
+
+Fedora 27 release extends Atomic Host availability from x86_64 to additional architectures which includes 64 bit ARM (AArch64) and PowerPC Little Endian (ppc64le).
+Atomic Host provides immutable operating system tree (OStree) with rollback facility. To learn more, see [Project Atomic website](http://www.projectatomic.io/).
+
+# Download
+Fedora Atomic Host for aarch64 and ppc64le are available in both ISOs and cloudImages-
+[ISO](link)
+[QCOW](link)
+[Raw image](link)
+
+# Installation
+Atomic Host can be installed as per user preference where they want to install it. For example: on bare metal, virtual machine, cloud setup, etc.
+One of the quick ways is to run `virt-install`
```

Comment:
Maybe we can add "If you have some aarch64 or ppc64le bare metal we'd love for you to take Atomic Host for a spin. One quick way is to run `virt-install` on one of theses platforms:"